infra/feast-operator/config/overlays/odh/kustomization.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: opendatahub
+
+
+resources:
+- .../default
+
+
+patches:
+  # patch to remove default `system` namespace in ../../manager/manager.yaml
+  - path: delete-namespace.yaml
+
+configMapGenerator:
+  - name:  feast-operator-parameters
+    envs:
+      - params.env
+
+configurations:
+  - params.yaml
+
+replacements:
+  - source:
+      kind: ConfigMap
+      name: feast-operator-parameters
+      version: v1
+      fieldPath: data.odh-feast-operator-controller-image
```

Comment:
what CM does this refer to?

Reply:
it's  generated from the params.env see the above. 

```
configMapGenerator:
  - name: feast-operator-parameters
    envs:
      - params.env
   ```
and then used in replacement to update the image in deployment. For example in the current deployment the image deployment will be like `image: 'docker.io/feastdev/feast-operator:0.42.0' instead `feastdev/feast-operator:0.42.0` not sure this answer you question.